lib/index.ts

```diff
@@ -135,6 +135,10 @@ class Y18N {
   }
 
   updateLocale (obj: Locale) {
+    if (this.locale === '__proto__') {
+      throw new Error('Prototype pollution attempt detected')
```

Comment:
Throwing an error doesn’t seem useful here; it seems like the proper fix would be to set a strong property instead of looking up the [[Prototype]].